src/shell/shell.cpp

```diff
@@ -251,28 +221,25 @@ void DOS_Shell::ParseLine(char *line)
 		line[0] = ' ';
 	line = trim(line);
 
+	constexpr bool fcb = true;
```

Comment:
You've done enough @weirddan455, so you're off the hook... But WTF is `fcb`? Job for the next guy who touches this file to figure out, i guess 😆

Reply:
https://en.wikipedia.org/wiki/File_Control_Block